test/extended/apiserver/health_endpoints.go

```diff
@@ -0,0 +1,108 @@
+package apiserver
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"strings"
+
+	"github.com/onsi/ginkgo/v2"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
+	restclient "k8s.io/client-go/rest"
+	"k8s.io/kubernetes/test/e2e/framework"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = ginkgo.Describe("[sig-api-machinery] API health endpoints", func() {
+	f := framework.NewDefaultFramework("api-health-endpoints")
+	f.SkipNamespaceCreation = true
+
+	oc := exutil.NewCLIWithoutNamespace("api-health-endpoints").AsAdmin()
+
+	ginkgo.It("should contain the required checks for the openshift-apiserver APIs", func() {
```

Comment:
Is it possible to skip this one in MicroShift? We do not have the openshift-apiserver components.

Reply:
sure, thanks for letting me know.

Reply:
done.